packages/internals/src/__tests__/engine-commands/queryEngineCommons.test.ts

```diff
@@ -0,0 +1,68 @@
+import { BinaryType } from '@prisma/fetch-engine'
+import * as E from 'fp-ts/Either'
+
+import { loadNodeAPILibrary } from '../../engine-commands/queryEngineCommons'
+import { resolveBinary } from '../../resolveBinary'
+import * as loadUtils from '../../utils/load'
+
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+
+describeIf(process.env.PRISMA_CLI_QUERY_ENGINE_TYPE === 'library')('loadNodeAPILibrary', () => {
```

Comment:
Would be better to run it by default even if the env var is not set 
```suggestion
describeIf(process.env.PRISMA_CLI_QUERY_ENGINE_TYPE !== 'binary')('loadNodeAPILibrary', () => {
```